app/models/like.rb

```diff
@@ -0,0 +1,14 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_post_likes_counter
+  after_create :update_post_likes_counter
+  after_destroy :update_post_likes_counter
```

Comment:
- [ ]  I suggest the like counter is only updated when a like is saved to the database, updating after creating and destroying a comment is unnecessary. :blush: